install_sct

```diff
@@ -703,6 +696,11 @@ if [[ -e "$RC_FILE_PATH" ]]; then
       print info "In case an old version SCT is already installed (4.0.0-beta.1 or before), remove 'sct_env' declaration in RC file"
       sed -ie '/sct_env/ s/^#*/#/' "$RC_FILE_PATH"
     fi
+    if grep "^export MPLBACKEND=Agg" "$RC_FILE_PATH"; then
+      print info "Commenting out 'export MPLBACKEND=Agg' from previous SCT installation"
+      export MPL_COMMENTS='# The line below has been commented out by a more recent installation of SCT,\n# because export MPLBACKEND=Agg is no longer needed as of SCT v5.3.1.\n'
+      sed -ie 's/^export MPLBACKEND=Agg/'"$MPL_COMMENTS"'# export MPLBACKEND=Agg/' "$RC_FILE_PATH"
```

Comment:
I've taken your suggestion @valosekj and added a little comment here just so that if anyone happens to revert back to their previous installation of SCT (by manually editing the RC file), they can understand why the line was commented out in the first place.

Reply:
Also, the `sct_check_dependencies` issue should be fixed now.

Reply:
I have tried this condition, but it looks that `sed` on MacOS converts the `\n` to an `n`. The result is then single long line:

```
# The line below has been commented out by a more recent installation of SCT,n# because export MPLBACKEND=Agg is no longer needed as of SCT v5.3.1.n# export MPLBACKEND=Agg
```

I found discussion about that [here](https://unix.stackexchange.com/a/152389)

Reply:
Thank you for testing this! One of the comments in the link you shared says:

> On OSX, you just need to add a $ before the sed string to prevent it from converting the \r to an r

So I'll give that a shot. :slightly_smiling_face:

Reply:
Sorry, I forgot to mention, that I tried the to play with `$`, but without success :-(

Reply:
Ack, sorry for the force-pushes. Hope it hasn't made working on this branch annoying. :sweat_smile:

Reply:
https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3401/commits/d5760db5e90cdf0d9245905b276a2fd13eb9bcb9 works fine! 🥳